### ISSUE TITLE
Embdedding layer visualization on Tensorboard

### ISSUE THREAD

**Issue by henry090** [rstudio/keras#567]
Hi @skeydan ,

If it is possible, then how to show  via this:

`callback_tensorboard(log_dir = NULL, histogram_freq = 0,
  batch_size = 32, write_graph = TRUE, write_grads = FALSE,
  write_images = FALSE, embeddings_freq = 0,
  embeddings_layer_names = NULL, embeddings_metadata = NULL,
  embeddings_data = NULL, update_freq = "epoch")`

embedding visualization of words/images on Tensorboard?

I read the documentation but still confused. What information I should provide for:

`embeddings_layer_names = NULL, embeddings_metadata = NULL,
  embeddings_data = NULL`

For example [IMDB text  classification](https://tensorflow.rstudio.com/keras/articles/tutorial_basic_text_classification.html).

`vocab_size <- 10000`

model <- keras_model_sequential()
model %>% 
  layer_embedding(input_dim = vocab_size, output_dim = 16) %>%
  layer_global_average_pooling_1d() %>%
  layer_dense(units = 16, activation = "relu") %>%
  layer_dense(units = 1, activation = "sigmoid")

What should be written in `callback_tensorboard` within `fit()` ?

BTW, I just can `save model`, `log` and see it on Tensorboard but don't know how to implement embedding visualization of keras embedding.

**Comment by skeydan** [comment]
We have an example here:

https://github.com/rstudio/keras/blob/master/vignettes/examples/mnist_cnn_embeddings.R

Does this help?

**Comment by henry090** [comment]
@skeydan of course, it helps a lot. But I want to see PCA on Tensorboard. Unfortunately, I got this:
<img width="1276" alt="screen shot 2018-10-16 at 23 53 35" src="https://user-images.githubusercontent.com/38523699/47043518-e01f8480-d19e-11e8-8ccd-5e4db8641d68.png">

**Comment by skeydan** [comment]
Okay, great you found out the reason!
